### DISCUSSION TITLE
BO order - Modules should be able to set default payment state

### DISCUSSION THREAD

**Discussion by Hlavtox** [PrestaShop/PrestaShop#41083] [Ideas]
#### Is your feature request related to a problem?

When creating BO order, employees do not know which order state to select when creating order in backoffice.

#### Describe the solution you'd like

A) Add a helpbox where we can hook what status should they select.
B) Or allow modules to select a default state Prestashop will select for their module.

#### Alternatives you've considered

I hardcoded a message box into the form, to provide help to customers.

![manual orders - kopie](https://user-images.githubusercontent.com/6097524/125455012-4090b08e-a26d-48f6-b632-4f3fdb5a0882.JPG)

**Comment by florine2623** [comment]
Hello @Hlavtox ,

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 

Keep in mind that there's more info about order status in the Help section :
<img width="303" alt="Screenshot 2021-07-13 at 18 11 06" src="https://user-images.githubusercontent.com/16019289/125487513-9edadca1-0c71-456b-9100-e1f3ae13a9cf.png">

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know!

Thank you

**Comment by florine2623** [comment]
Ping @PrestaShop/product-team ,

What do you think about @Hlavtox 's idea ?

Thanks!

**Comment by fire2** [comment]
Also interested in this - small details make the difference.

What I currently do in my shops is load a JS file in this view that changes the order status dropdown value automatically when the user changes the payment dropdown selection. This way I don't hardcode anything on the template, but it's still not an ideal solution. This should be part of the core, since every payment module has it's own order status.

**Comment by Hlavtox** [comment]
cc @kpodemski @Codencode
